API.md

```diff
@@ -77,7 +77,11 @@ Parameters
     flags: the flags of the encrypted document (optional)
     documentHash: the hash of the encrypted document (optional)
     nonce: the nonce of the encrypted document (required)
-    signature: the signature of the encrypted document (required)
+    signature: the signature of the encrypted document (required).
+     The signature is based on hashing the following parameters:
+     transactionId + dataNftAddress + decrypterAddress + chainId + nonce
```

Comment:
How is the message hashed ?
is dataNftAddress lowercase? checksum format?
is this a string concatenation ? ( transactionId + dataNftAddress + decrypterAddress + chainId + nonce) or is a bytes sum?